### PR TITLE
Update docs about post creation

### PR DESCRIPTION
**Pull request by 104fps**

### CODE REVIEW

**Comment by pathawks** [comment]
Fixes #7137

File: docs/_docs/posts.md [hunk 1]
```diff
@@ -122,7 +122,11 @@ categories: jekyll update
 
 You’ll find this post in your `_posts` directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run `bundle exec jekyll serve`, which launches a web server and auto-regenerates your site when a file is updated.
 
-To add new posts, simply add a file in the `_posts` directory that follows the convention `YYYY-MM-DD-name-of-post.ext` and includes the necessary front matter. Take a look at the source for this post to get an idea about how it works.
+Jekyll requires blog post files to be named according to the following format:
+
+`YEAR-MONTH-DAY-title.MARKUP`
+
+Where `YEAR` is a four-digit number, `MONTH` and `DAY` are both two-digit numbers, and `MARKUP` is the file extension representing the format used in the file. After that include the necessary front matter. Take a look at the source for this post to get an idea about how it works.
```
Comment (pathawks) [inline docs/_docs/posts.md]: This content is already present in the page (lines 26-35), so I'm not sure it makes sense to duplicate it here, though I do understand that this is simply a reflection of the default post in the site template.

File: test/source/_encodings/UTF8CRLFandBOM.md [hunk 2]
```diff
@@ -8,4 +8,8 @@ This file was created with CR/LFs, and encoded as UTF8 with a BOM
 
 You’ll find this post in your `_posts` directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run `bundle exec jekyll serve`, which launches a web server and auto-regenerates your site when a file is updated.
 
-To add new posts, simply add a file in the `_posts` directory that follows the convention `YYYY-MM-DD-name-of-post.ext` and includes the necessary front matter. Take a look at the source for this post to get an idea about how it works.
+Jekyll requires blog post files to be named according to the following format:
+
+`YEAR-MONTH-DAY-title.MARKUP`
+
+Where `YEAR` is a four-digit number, `MONTH` and `DAY` are both two-digit numbers, and `MARKUP` is the file extension representing the format used in the file. After that include the necessary front matter. Take a look at the source for this post to get an idea about how it works.
```
Comment (pathawks) [inline test/source/_encodings/UTF8CRLFandBOM.md]: We probably don't want/need to modify our `test` site.

File: docs/_docs/posts.md (hunk 1 above)
Comment (104fps) [reply docs/_docs/posts.md]: Ah, my bad, agreed with you, just keep this update in _post template

File: lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb [hunk 3]
```diff
@@ -6,7 +6,11 @@ categories: jekyll update
 ---
 You’ll find this post in your `_posts` directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run `jekyll serve`, which launches a web server and auto-regenerates your site when a file is updated.
 
-To add new posts, simply add a file in the `_posts` directory that follows the convention `YYYY-MM-DD-name-of-post.ext` and includes the necessary front matter. Take a look at the source for this post to get an idea about how it works.
+Jekyll requires blog post files to be named according to the following format:
+
+`YEAR-MONTH-DAY-title.MARKUP`
+
+Where `YEAR` is a four-digit number, `MONTH` and `DAY` are both two-digit numbers, and `MARKUP` is the file extension representing the format used in the file. After that include the necessary front matter. Take a look at the source for this post to get an idea about how it works.
```
Comment (chrisfinazzo) [inline lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb]: Being less explicit here is not something I thought would be an issue, but here we are 🤷‍♂️ .

Welcome to the project, @104fps. Only 1 other change and this is 👍🏼.

"After that, include the necessary front matter."

**Comment by 104fps** [comment]
@chrisfinazzo Thanks, appreciate it! 

Updated

**Comment by pathawks** [comment]
@jekyllbot: merge +docs

**Comment by pathawks** [comment]
@104fps Thank you much for this :beers:

**Pull request merged**